### PR TITLE
Add Inactive account endpoint

### PR DESCRIPTION
**Pull request by malosayli**
# Inactive Account Management

## Changes Made
- Added configuration options for account inactivity management:
  - Inactivity threshold period configuration
  - Custom schema modifications to support new settings
- Implemented new API endpoints:
  - PUT endpoint to list inactive identities and deactivate them
- Added database layer support:
  - SQL queries for identifying inactive accounts based on last activity
  - Update operations for account deactivation
- Implemented business logic:
  - Manager methods for inactive account detection
  - Persister methods for handling deactivation states
  - Activity tracking mechanisms

## Why
To enhance security and resource management by automatically identifying and handling dormant accounts. This feature allows administrators to:
- Monitor account activity patterns
- Enforce account lifecycle policies
- Maintain cleaner user databases
- Reduce security risks from abandoned accounts

## Technical Details
- Extended Kratos schema to include inactivity configuration
- Added persistence layer modifications for tracking last activity
- Implemented RESTful endpoints for administrative control

### CODE REVIEW

**Comment by malosayli** [comment]
@aalkhodiry For the schema version source I was able to point to nayla version by adding this comment in the top of the file 
```yaml
# yaml-language-server: $schema=https://raw.githubusercontent.com/nayla-finance/kratos/refs/heads/master/.schema/version.schema.json
```

File: identity/handler.go [hunk 1]
```diff
@@ -1023,3 +1028,72 @@ func (h *Handler) deleteIdentityCredentials(w http.ResponseWriter, r *http.Reque
...
+	identities := make([]map[string]interface{}, len(inactiveIdentities))
+
+	if len(inactiveIdentities) > 0 {
+		ids := make([]string, len(inactiveIdentities))
+		for i, id := range inactiveIdentities {
+			identities[i] = map[string]interface{}{
+				"id":         id.ID,
+				"state":      id.State,
+				"created_at": id.CreatedAt,
+				"updated_at": id.UpdatedAt,
```
Comment (aalkhodiry) [inline identity/handler.go:1087]: I think this should include a last activity field some how (like `last_session_at`)

**Review by aalkhodiry — APPROVED** [review]

File: identity/handler.go (hunk 1 above)
Comment (malosayli) [reply identity/handler.go:1087]: Yes I agree it'll be easier to add a `last_active_at` to identities table and update it on every login which will make the lookup for inactive much easier. with this we don't need to keep at least one session for each identity we can just run a job to delete all the sessions every few days

Comment (aalkhodiry) [reply identity/handler.go:1087]: No not like this, I mean last_active_at is the max session time in the query

**Pull request merged**